### DISCUSSION TITLE
怎么在树莓派(arm)用docker 部署,文档的是amd的,我对docker不算太懂 有模板吗

### DISCUSSION THREAD

**Discussion by mrchenxxx** [nonebot/discussions#116] [问答]
怎么在树莓派(arm)用docker 部署,文档的是amd的,我对docker不算太懂 有模板吗

**Comment by yanyongyu** [comment]
文档里使用的docker镜像都是支持arm的，只要不是alpine镜像应该就行

或者你可以参考 
https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker
https://fastapi.tiangolo.com/deployment/docker/ 搭建自己的镜像

**Reply by mrchenxxx** [reply]
![QQ图片20220227202528](https://user-images.githubusercontent.com/44868578/155882254-01002d73-4445-4452-8eeb-86a00429f714.png)
这个错误咋解决

**Reply by yanyongyu** [reply]
参考后面的fastapi文档自行构建一个python镜像
